subprojects/tracy.wrap

```diff
@@ -0,0 +1,3 @@
+[wrap-git]
+url = https://github.com/wolfpld/tracy.git
+revision = 9b15870991da9ebf7d4e2b808691c3ce91af9356
```

Comment:
I tried building this on MinGW, but it fails with:

```
../subprojects/tracy/client/TracySysTrace.cpp: In function ‘void tracy::SetupVsync()’:
../subprojects/tracy/client/TracySysTrace.cpp:303:5: error: ‘EVENT_FILTER_EVENT_ID’ was not declared in this scope; did you mean ‘EVENT_FILTER_HEADER’?
  303 |     EVENT_FILTER_EVENT_ID fe = {};
      |     ^~~~~~~~~~~~~~~~~~~~~
      |     EVENT_FILTER_HEADER
../subprojects/tracy/client/TracySysTrace.cpp:304:5: error: ‘fe’ was not declared in this scope
  304 |     fe.FilterIn = TRUE;
      |     ^~
../subprojects/tracy/client/TracySysTrace.cpp:311:17: error: ‘EVENT_FILTER_TYPE_EVENT_ID’ was not declared in this scope; did you mean ‘EVENT_FILTER_TYPE_SCHEMATIZED’?
  311 |     desc.Type = EVENT_FILTER_TYPE_EVENT_ID;
      |                 ^~~~~~~~~~~~~~~~~~~~~~~~~~
      |                 EVENT_FILTER_TYPE_SCHEMATIZED
../subprojects/tracy/client/TracySysTrace.cpp:315:29: error: ‘EVENT_ENABLE_PROPERTY_IGNORE_KEYWORD_0’ was not declared in this scope
  315 |     params.EnableProperty = EVENT_ENABLE_PROPERTY_IGNORE_KEYWORD_0;
      |                             ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Reply:
PR has been merged.

Reply:
```
Cloning into 'tracy'...
remote: Enumerating objects: 31151, done.
remote: Counting objects: 100% (7416/7416), done.
remote: Compressing objects: 100% (255/255), done.
remote: Total 31151 (delta 7198), reused 7284 (delta 7153), pack-reused 23735
Receiving objects: 100% (31151/31151), 17.89 MiB | 17.60 MiB/s, done.
Resolving deltas: 100% (22675/22675), done.
HEAD is now at 9b158709 Merge pull request #361 from Juice-Labs/meson-support

Executing subproject tracy 


../subprojects/tracy/meson.build:1:0: ERROR: In subproject tracy: Unknown options: "tracy:tracy_no_sys_trace"

```
Still getting this when trying to build.